packages/elements/src/components/card/__styles__/index.ts

```diff
@@ -71,9 +71,7 @@ export const ElCardSubHeading = styled.h6`
   }
 `
 
-export const elCardSubHeadingAdditionalExpanded = css`
-  margin-bottom: 1.25rem;
-`
+export const elCardSubHeadingAdditionalExpanded = css``
```

Comment:
can be removed? Not an element so presuming it's an additional condition state?

Reply:
It's left in to avoid breaking changes in case the class is being used. It now no longer does anything as the component is expanded by default